src/WebJobs.Script/Rpc/RpcInitializationService.cs

```diff
@@ -68,7 +68,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
         public Task StopAsync(CancellationToken cancellationToken)
         {
             _logger.LogDebug("Shuttingdown Rpc Channels Manager");
-            _webHostlanguageWorkerChannelManager.ShutdownChannels();
+            _webHostlanguageWorkerChannelManager.ShutdownChannelsAsync();
```

Comment:
>_webHostlanguageWorkerChannelManager.ShutdownChannelsAsync(); [](start = 12, length = 61)

await? and StopAsync could be an async Task I think

Reply:
Good catch! Thanks!